presto-feature-toggle/src/main/java/com/facebook/presto/features/config/FileBasedFeatureToggleModule.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.features.config;
+
+import com.facebook.presto.features.binder.PrestoFeatureToggle;
+import com.facebook.presto.features.http.FeatureToggleInfo;
+import com.facebook.presto.features.strategy.FeatureToggleStrategyFactory;
+import com.google.inject.Binder;
+import com.google.inject.Inject;
+import com.google.inject.Module;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+
+import static com.facebook.airlift.jaxrs.JaxrsBinder.jaxrsBinder;
+import static com.facebook.presto.features.config.ConfigurationParser.parseConfiguration;
+import static com.google.common.base.Suppliers.memoizeWithExpiration;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+public class FileBasedFeatureToggleModule
```

Comment:
Is this unused?

Reply:
`FileBasedFeatureToggleModule` is a FeatureToggle module for file-based configuration.
For other configuration sources, we can create and initialize other modules. for example: `DatabaseBasedFeatureToggleModule`.

I'll try to refactor this and extract the configuration sources parser in a separate class, and we will have a `FeatureToggleModule` and configuration parsers based on configuration sources